site/src/components/banner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Banner
+description: The Banner component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fnode-id%3D1002%253A7106%26t%3DWU64bU98uaT1nU7A-1
+classType: Component
+classPrefixes:
+  - banner
+---
+
+# Banner
+
+Banner component allows adding a full width notification block to the page.
```

Comment:
Can we provide a paragraph here that briefly explains the difference between the Banner component and the Notification component, and then links out to the Notification component page?

These two components seem pretty similar to me, and I think it'd be good to make the difference between them evident up front.

Reply:
They are similar but for sure different ones. I've updated the description a bit on the banners page. let me know if you have any suggestions. Words are not my strongest side :D And I don't think these two need to be linked to each other, maybe only if we add a sentence like that - "For alerting users of something within the content we'd recommend using the notification component (link)".